tools/scripts/lib_install/fmtlib.sh

```
@@ -0,0 +1,84 @@
+make -j${num_cpus}
+
+# Check if checkinstall is installed
+set +e
+command -v checkinstall
+checkinstall_installed=$?
+set -e
+
+# Install
+install_command_prefix="${privileged_command_prefix}"
+if [ $checkinstall_installed -eq 0 ] ; then
+  install_command_prefix="${install_command_prefix} checkinstall --pkgname '${package_name}' --pkgversion '${version}' --provides '${package_name}' --nodoc -y --pakdir \"${deb_output_dir}\""
```

Shellcheck

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)